docs/index.rst

```diff
@@ -65,6 +65,20 @@ library status.
 - :py:mod:`dataclasses` contains classes for normalizing data (e.g. an
   ``IVCurve`` class)
 
+Dependencies
+------------
+
+This project follows the guidelines laid out in NEP-29. It supports:
+
+- All minor versions of Python released 42 months prior to the project,
+  and at minimum the two latest minor versions.
+- All minor versions of numpy released in the 24 months prior to the project,
+  and at minimum the last three minor versions
+
```

Comment:
might be worth specifying a pvlib dependency policy, even if it's not set in stone.

Reply:
Not sure what the best policy is for pvlib. So far I have tended to track the latest version pretty closely rather than try to support across minor versions with breaking API differences.

Reply:
For now I think dependency on the latest release of pvlib (v0.8) is okay. We can test pvanalytics for pvlib >= v0.8 and forward.